apps/docs/src/app/core/component-docs/timeline/examples/timeline-basic-example/timeline-basic-example.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="timeline-example-container">
+    <div class="timeline-example-container__col timeline-example-container__col--left">
+        <fd-timeline [dataSource]="data" [trackBy]="trackBy">
+
+            <fd-timeline-node *fdTimelineNodeDef="let node">
+
+                <fd-timeline-node-header>
+                    <fd-avatar [circle]="true" size="xs" [label]="node.user.name" [image]="node.user.avatar"></fd-avatar>
+                    <fd-timeline-node-header-info>
+                        <span fdTimelineNodeHeaderInfoTitle>{{node.user.name}}</span>
```

Comment:
while we give the user the 2 options, we use kebab-case in our documentation. Please change it, because people will start using both and we will end up with an inconsistent documentation